test/mCODEPrimaryCancerCondition.test.js

```diff
@@ -1,8 +1,5 @@
 const path = require('path');
-// eslint-disable-next-line object-curly-newline
 const { defaultLoadElm, loadJSONFixture, defaultLoadValuesets, mapValueSets, execute } = require('cql-testing-harness');
-// eslint-disable-next-line import/order
-const { Date } = require('cql-execution/lib/datatypes/datetime');
 
 let executionResults;
 beforeAll(() => {
```

Comment:
DP2 - Small consistency change plz

```suggestion
beforeAll(() => {
  // Set up necessary data for cql-execution
```